Makefile

```diff
@@ -7,12 +7,14 @@ export GO111MODULE=on
 export GOFLAGS?=-mod=readonly -trimpath
 export GIT_TAG ?= $(shell git tag --points-at HEAD)
 
-GO_VERSION = 1.20.1
+GO_VERSION = 1.19
```

Comment:
What needs 1.19?

Reply:
I had some issues with the controller-tools, but I guess I fixed it in another project. Will reflect the same fix here.